<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportTestDlg</class>
 <widget class="QWidget" name="ExportTestDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTabWidget" name="ktabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Task definition:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="leTitle"/>
       </item>
       <item row="2" column="1">
        <widget class="KTextEdit" name="teTaskDefinition"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>System definition:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Output format:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Options:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="cbTables">
         <property name="text">
          <string>Tables</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="cbGraphs">
         <property name="text">
          <string>Graphs</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Conclusion:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="KTextEdit" name="teConclusion"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbOutputFormat"/>
       </item>
       <item row="3" column="1">
        <widget class="KTextEdit" name="teSystemDefinition"/>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="leTag"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Tag:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Vocabulary</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Tag:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="leVocabularyTag"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Notes:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KTextEdit" name="teVocabularyNotes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Grammar</string>
      </attribute>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>30</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
      <widget class="KLineEdit" name="leGrammarTag">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>3</y>
         <width>436</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>3</y>
         <width>25</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tag:</string>
       </property>
      </widget>
      <widget class="KTextEdit" name="teGrammarNotes">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>436</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Training</string>
      </attribute>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>50</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
      <widget class="KLineEdit" name="leTrainingTag">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>23</y>
         <width>436</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>23</y>
         <width>25</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tag:</string>
       </property>
      </widget>
      <widget class="KTextEdit" name="teTrainingNotes">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>436</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
